### ISSUE TITLE
Discuss and find a replacement for Collab news feed carousel

### ISSUE THREAD

**Issue by Troy-Lawson** [gctools-outilsgc/gccollab#298]
Failures to the Success Criteria: 1.4.3 Contrast (Minimum) (Level AA)
1. Text over image in carousel does not meet contrast minimum, example (“Free to be me
| Sois toi-meme”). Minimum contrast for large text is 3:1.

2.2.2 Pause, Stop, Hide (Level A)
1. Newsfeed page has a carousel that does not have controls to pause, stop or play.

Failures to the Success Criteria: 2.4.7 Focus Visible (Level AA)
1. Focus on Newsfeed carousel controls are large. Focus on right slide control covers both
left and right slide control buttons, making focus selection unclear.

**Comment by markwooff** [comment]
Carousel on collab is very useful for showcasing groups within the platform. Could we find an accessible JS library to replace it with? Or an alternative way to showcase groups on the home page.

**Comment by mgifford** [comment]
Well, there are always sites like http://shouldiuseacarousel.com/
or https://www.nngroup.com/articles/designing-effective-carousels/ 
that suggest that it isn't a great solution. 

My approach would be to load one frame by default.  Allowing people to navigate between frames if they want to.

**Comment by markwooff** [comment]
Yeah that http://shouldiuseacarousel.com/ site always comes up, and I usually agree with it. Would like to keep the ability to showcase multiple groups on the homepage but understand it's an accessibility issue. Your suggestion would work!

**Comment by mgifford** [comment]
Generally I find carousels are most useful for internal politics.  If you can provide an option for a random support so that everyone gets featured & everyone can browse the features then you can more easily deal with the complexity of usability/accessibility problems and insure that people feel like their pet project is being properly promoted. 

Also worth noting that if WET had a pattern for this, I don't think that they are regularly testing to see that it works in browsers/AT.  I could be wrong about that.

**Comment by LemieuxGen** [comment]
Carousel is nice, but we shouldn't be using it if it's not accessible. What about randomized images (with caption) like what is done on Canada.ca? the images changes every time you refresh. https://www.canada.ca/en.html

**Comment by markwooff** [comment]
@LemieuxGen love that idea! Solves both the accessibility issue and the fact that we'd want to showcase more than one group on the home page.

**Comment by markwooff** [comment]
@LemieuxGen also noticed that we're featuring some groups on the main page of the wiki. Wondering if it's worth making changes to the carousel on GCcollab vs. just using the wiki going forward. Thoughts?

**Comment by LemieuxGen** [comment]
@markwooff I think there's value in promoting communities on the wiki, and also having images on collab that showcase great initiatives. The wiki doesn't have enough trackion yet to use it as our main place to feature communities/initiatives. I'll speak with the outreach team to get their view on this.
